### ISSUE TITLE
NO ISSUE JUST Question.

### ISSUE THREAD

**Issue by luckas72** [martyngal/anki-ipa#20]
First, thank you for devoting time to improve Anki community's experience. Hmm, the question that I have, does the code only proceed phonetic annotation when it is a translation? I tried to get the English phonetic of some words written in English from a field, and it only shows the word itself. Then, when I tried to get the phonetic of a French word from a word written in English from a field, and it worked. So is that the original purpose of the add on? Or is that an unexpected thinggy?

**Comment by martyngal** [comment]
Which words did you try to transcribe exactly?

**Comment by luckas72** [comment]
I used nostrum, neophyte, and necromancy. The IPA shows the exact same word,  when it was english to english, but shows IPA when it was english to french.

**Comment by martyngal** [comment]
I get the following results using the newest Anki IPA version: 

for English:
- nostrum -> nostrum*
- neophyte -> ˈniəˌfaɪt
- necromancy -> ˈnɛkrəˌmænsi

for French:
- nostrum -> nɒs.trəm
- neophyte -> IPA not found
- necromancy -> IPA not found

There exists an inconsistency when IPA transcriptions aren't found. For English the add-on returns the input with a star at the end, for every other language it shows the message "IPA not found." I will work on that in the future.

**Comment by luckas72** [comment]
Yeah, that is weird. When I do mine, it shows the same thing for neophyte for example:

![image](https://user-images.githubusercontent.com/44622758/89806595-49575100-db05-11ea-996d-b96e4e512b82.png)
![image](https://user-images.githubusercontent.com/44622758/89806609-4fe5c880-db05-11ea-9fab-a365e361e854.png)
